go.mod

```diff
@@ -35,9 +36,7 @@ require (
 	github.com/gravitational/configure v0.0.0-20160909185025-1db4b84fe9db
 	github.com/gravitational/form v0.0.0-20151109031454-c4048f792f70
 	github.com/gravitational/kingpin v2.1.11-0.20190130013101-742f2714c145+incompatible
-	github.com/gravitational/license v0.0.0-20180912170534-4f189e3bd6e3
 	github.com/gravitational/oxy v0.0.0-20200916204440-3eb06d921a1d
-	github.com/gravitational/reporting v0.0.0-20180907002058-ac7b85c75c4c
```

Comment:
these changes don't seem right, but they happened when I ran "make update-vendor" to include `survey`.

Reply:
I think this is because you don't have the Enterprise source available to you.